cmd/status.go

```diff
@@ -160,9 +160,9 @@ func getRole(pid int) (role string, err error) {
 	}
 	cmdln := string(raw)
 	if strings.Contains(cmdln, "enable-worker") {
-		return "server+worker", nil
-	} else if strings.Contains(cmdln, "server") {
-		return "server", nil
+		return "controller+worker", nil
```

Comment:
Just leaving this note here so we don't forget to refactor k0sctl since it is reading this value as server+worker